classification1.Rmd

```diff
@@ -425,20 +442,20 @@ cancer |>
 ```
```

Comment:
right above this line

"subset the first 5 rows"

I'm not a fan of using "subset" as a verb

Reply:
I see some instances of this below too, FYI